test/controllers/posts_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class PostsControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
+    get posts_index_url
+    assert_response :success
+  end
+
+  test 'should get show' do
+    get posts_show_url
+    assert_response :success
+  end
+end
```

Comment:
- I appreciate you writing the test as well, as it's not required for this milestone. :clap: :clap: